README.md

```diff
@@ -22,17 +21,51 @@ to the sites' source code.
 
 ## Install
 To evaluate the software, we recommend installing from the [latest Open ONI
-release](https://github.com/open-oni/open-oni/releases/latest).
+release](https://github.com/open-oni/open-oni/releases).
 
 The 1.0 release includes upgrading Django to the latest LTS 2.2 release along
 with migrating to Python 3 and upgrading Solr and MariaDB to more recent
-releases. Note that release 0.11 and prior run Python 2 and Django 1.11 LTS which goes out of support in April 2020.
+releases. Note that release 0.11 and prior run Python 2 and Django 1.11 LTS
+which goes out of support in April 2020.
 
 If you're interested in more recent changes, check out the `dev` branch.
 
+### Resource Requirements
+
+This is a ballpark estimate of resources we recommend for running Open ONI in
+production. Open ONI is quite capable with minimal processing and memory
+resources.
+
+CPU | Memory | Storage (with TIFFs) | Storage (JP2s only)
+----|--------|----------------------|--------------------
+Modern CPU with two (virtual) cores | 6GB | 1TB per ~25K pages| 1TB per ~100K pages
+
+We share more information on the [Resource
+Usage](https://github.com/open-oni/open-oni/wiki/Resource-Usage) page in our
+wiki, which may further inform your resource allocation.
+
+### Dependency Roadmap
+
+We expect Open ONI's major components (aside from Bootstrap 3) to be supported
+until the next Django LTS release 3.2 in April 2021, at which point we will
+likely upgrade Open ONI's components between major releases again.
+
+Component | Version | Supported Until | Next Target Release | Documentation
+----------|---------|-----------------|---------------------|--------------
+Python | 3.6 | 2021-12 | TBD | https://www.python.org/dev/peps/pep-0494/#lifespan
+Django | 2.2 LTS | 2022-04 | 3.2 LTS, 2021-04 | https://www.djangoproject.com/download/#supported-versions
+MariaDB | 10.4 | 2024-06 | TBD | https://mariadb.com/kb/en/mariadb-server/
+Solr | 8.x | Approx 2022, based on ~18 major release cycle | TBD | https://lucene.apache.org/solr/downloads.html#about-versions-and-support
+jQuery | 3.4.1 | Unknown | TBD | https://github.com/jquery/jquery/wiki/Roadmap
+Bootstrap | 3.4.1 | 2019-07-24 | TBD | https://github.com/twbs/release
+OpenSeadragon | 2.4.1 | Unkown | TBD | https://github.com/openseadragon/openseadragon
+tablesorter | 2.31.2 | Unknown | TBD | https://github.com/mottie/tablesorter/
+
 ## Documentation
-- [Installation, Configuration, and Troubleshooting](https://github.com/open-oni/open-oni/tree/master/docs)
-- [Development Standards, General Information](https://github.com/open-oni/open-oni/wiki)
+- [Installation, Configuration, and
+  Troubleshooting](https://github.com/open-oni/open-oni/tree/master/docs)
```

Comment:
Do we offer troubleshooting in the docs?  I'm not sure that we do explicitly, anyway, but if you mean in a more general sense then this is okay

Reply:
I don't remember writing that, but yeah I think it's okay for it to lead people there in a more general sense.

Oops. Now that I see this little snippet, I'll want to remove that label saying "Development Standards" are in the wiki, because they've moved to CONTRIBUTING.md. But I'll look at something to describe the other docs in there, like the institutions' resource usage doc I added :thinking: